test/doctest/source-highlightjs-languages.html

```diff
@@ -0,0 +1,469 @@
+    // Use this method for navigation when auto-sliding
+    autoSlideMethod: Reveal.navigateNext,
+    // Specify the average time in seconds that you think you will spend
+    // presenting each slide. This is used to show a pacing timer in the
+    // speaker view
+    defaultTiming: 120,
+    // Enable slide navigation via mouse wheel
+    mouseWheel: false,
+    // Hides the address bar on mobile devices
+    hideAddressBar: true,
+    // Opens links in an iframe preview overlay
+    // Add `data-preview-link` and `data-preview-link="false"` to customise each link
+    // individually
+    previewLinks: false,
+    // Transition style (e.g., none, fade, slide, convex, concave, zoom)
+    transition: 'slide',
+    // Transition speed (e.g., default, fast, slow)
+    transitionSpeed: 'default',
+    // Transition style for full page slide backgrounds (e.g., none, fade, slide, convex, concave, zoom)
+    backgroundTransition: 'fade',
+    // Number of slides away from the current that are visible
+    viewDistance: 3,
+    // Parallax background image (e.g., "'https://s3.amazonaws.com/hakim-static/reveal-js/reveal-parallax-1.jpg'")
+    parallaxBackgroundImage: '',
+    // Parallax background size in CSS syntax (e.g., "2100px 900px")
+    parallaxBackgroundSize: '',
+    // Number of pixels to move the parallax background per slide
+    // - Calculated automatically unless specified
+    // - Set to 0 to disable movement along an axis
+    parallaxBackgroundHorizontal: null,
+    parallaxBackgroundVertical: null,
+    // The display mode that will be used to show slides
+    display: 'block',
+  
+    // The "normal" size of the presentation, aspect ratio will be preserved
+    // when the presentation is scaled to fit different resolutions. Can be
+    // specified using percentage units.
+    width: 960,
+    height: 700,
+  
+    // Factor of the display size that should remain empty around the content
+    margin: 0.1,
+  
+    // Bounds for smallest/largest possible scale to apply to content
+    minScale: 0.2,
+    maxScale: 1.5,
+  
+    // PDF Export Options
+    // Put each fragment on a separate page
+    pdfSeparateFragments: true,
+    // For slides that do not fit on a page, max number of pages
+    pdfMaxPagesPerSlide: 1,
+  
+    // Optional libraries used to extend on reveal.js
+    dependencies: [
+        { src: 'reveal.js/lib/js/classList.js', condition: function() { return !document.body.classList; } },
+        { src: 'reveal.js/plugin/zoom-js/zoom.js', async: true },
+        { src: 'reveal.js/plugin/notes/notes.js', async: true },
+  
+  
+  
+  
+    ],
+  
+  
+  
+  });
+</script><script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/yaml.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/scilab.min.js"></script><script>
```

Comment:
we can see that `languages/yaml.min.js` and `languages/scilab.min.js` are present.